docker/exporter/export_runner.py

```diff
@@ -74,6 +78,39 @@ def spawn_ecosystem_exporter(work_dir: str, bucket: str, eco: str):
     logging.error('Export of %s failed with Exit Code: %d', eco, return_code)
 
 
+def aggregate_all_vulnerabilities(work_dir: str, export_bucket: str):
+  """
+  Aggregates vulnerability records from each ecosystem into a single zip
+  file and uploads it to the export bucket.
+  """
+  zip_file_name = 'all.zip'
+  output_zip = os.path.join(work_dir, zip_file_name)
+  all_vulns = {}
+
+  for ecosystem_dir, _, files in os.walk(work_dir):
```

Comment:
A cleaner way to do this could be with python globs. https://docs.python.org/3/library/glob.html (See examples at the bottom). a glob of `**/*.json` should select all the files you need without additional if statements.

Reply:
(Make sure to set recursive to true) `glob.glob("**/*.json", recursive=True)`

Reply:
updated